panel/pane/image.py

```diff
@@ -344,6 +344,26 @@ def _imgshape(cls, data):
         return int(w), int(h)
 
 
+class JPEG(JPG):
+    """
+    The `JPEG` pane embeds a .jpeg image file in a panel if provided a
+    local path, or will link to a remote image if provided a URL.
+
+    Reference: https://panel.holoviz.org/reference/panes/JPEG.html
```

Comment:
I don't know if having a separate class for this is the _best_ approach, but it's what I thought of as well when I saw the original issue. That said, this link points to nowhere, so if other people think this is the way to go, I think the docstring should just say something like "Same as pn.pane.JPG, but accepting an extension .jpeg" and not trying to have a separate reference page.